timemachine/fe/chiral_utils.py

```diff
@@ -145,7 +145,8 @@ def find_chiral_atoms(mol):
     chiral_patterns = [
         "[X4:1]",  # any tetrahedral atom
         "[#16X3,#15X3:1]",  # trivalent sulfur, phosphorous are assumed to be non-invertible
-        # "[#7X3:1](~[R])(~[R])~[R]",  # nitrogen directly bonded to three ring atoms  # TODO: handle pyramidal nitrogen
+        "[#7X3:1](~F)(~F)~F",  # highly explosive, non-invertible nitrogen used for testing
```

Comment:
```suggestion
        "[#7X3:1](~F)(~F)~F",  # for software testing
```

(tangent: comment maybe referring to a different nitrogen trihalide? https://en.wikipedia.org/wiki/Nitrogen_trifluoride )